force-app/main/default/classes/Trigger Recipes/LogTriggerHandler.cls

```diff
@@ -0,0 +1,32 @@
+public with sharing class LogTriggerHandler extends TriggerHandler {
```

Comment:
use `inherited sharing` so that no sharing inforced here and class runs in System mode by default.

Reply:
done